builds/cmake/Modules/ConfigureWindows.cmake

```diff
@@ -43,6 +43,8 @@ if(MSVC)
 
 	option(MSVC_MULTICORE "MSVC: Build using multiple cores (/MP)" ON)
 	if (MSVC_MULTICORE)
-		add_compile_options(/MP)
+		add_compile_options("/MP")
 	endif()
+
+	add_compile_options("/utf-8")
```

Comment:
What I dislike here, is that the comment is lost.

Reply:
Valid concern, maybe make the comment more generic now "Interpret character literals as UTF-8"

Reply:
Added comment in #2017